### PR TITLE
Add thumbnail attribute to tweet object

### PR DESCRIPTION
**Pull request by RtiM0**
Fetch the thumbnail of a video in a tweet

### CODE REVIEW

**Comment by zinary** [comment]
#peru sar
# #  how are you so peru
> Teach me

**Comment by pielco11** [comment]
Missing support for:

- [ ] Pandas
- [ ] SQLite
- [ ] CSV
- [ ] JSON
- [ ] Elasticsearch

**Comment by RtiM0** [comment]
> Missing support for:
> 
> * [ ]  Pandas
> * [ ]  SQLite
> * [ ]  CSV
> * [ ]  JSON
> * [ ]  Elasticsearch

Last commit adds support for SQLite, CSV, JSON and Elasticsearch. I have not updated Pandas because similar data fields: URLs, Photos and Videos are already missing for some reason.

File: twint/storage/db.py [hunk 1]
```diff
@@ -265,6 +266,7 @@ def tweets(conn, Tweet, config):
                     ",".join(Tweet.cashtags),
                     ",".join(Tweet.urls),
                     ",".join(Tweet.photos),
+                    ",".join(Tweet.thumbnail)
```
Comment (pielco11) [inline twint/storage/db.py:269]: Here is missing a `,` at the end of the row

Comment (pielco11) [reply twint/storage/db.py:269]: And below at `cursor.execute` is missing a `,?` too, since there's a new field

**Comment by pielco11** [comment]
Thanks for the updates; about Pandas, I guess it's not a must also people can load from JSON/CSV so there's still a way

If you want to add the few missing fields, it'd be great otherwise don't worry about that. (Anyway is just a dict so not that hard)

File: twint/storage/db.py (hunk 1 above)
Comment (RtiM0) [reply twint/storage/db.py]: Embarrassing! it wouldn't require `join()` either since its not a list. Good catch.

**Comment by RtiM0** [comment]
I made an amend to last commit with that `db.py` fix and also updated `elasticsearch/index-tweets.json`.
I've also made a update.py commit to add the missing data fields. (which should solve #737)

**Pull request merged**